recipes/basis_set_exchange/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "basis_set_exchange" %}
+{% set version = "0.8.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 78e2cf3d9b6862e82b27e019c09812e01d40b785014a3969fd51b52079c90122
+
+build:
+  noarch: python
+  number: 0
+  script: "python -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
Make `python` -> `python >=3.5` under `host` and `run`.

Reply:
Done